docs/en_us/internal/testing.rst

```diff
@@ -196,6 +196,16 @@ To run a single test format the command like this.
 
     paver test_system -t lms/djangoapps/courseware/tests/tests.py:ActivateLoginTest.test_activate_login
 
+It is also helpful to be able to speed up tests by skipping Django migrations during test development. To make test
```

Comment:
It'd be great to wrap these lines at 80 chars, though I know the rest of the file doesn't do it.